Source/WebCore/dom/Element.cpp

```diff
@@ -450,7 +450,7 @@ static ShouldIgnoreMouseEvent dispatchPointerEventIfNeeded(Element& element, con
     if (RefPtr page = element.document().page()) {
         auto& pointerCaptureController = page->pointerCaptureController();
 #if ENABLE(TOUCH_EVENTS)
-        if (platformEvent.pointerId() != mousePointerID && mouseEvent.type() != eventNames().clickEvent && pointerCaptureController.preventsCompatibilityMouseEventsForIdentifier(platformEvent.pointerId()))
+        if (platformEvent.pointerId() != mousePointerID && mouseEvent.type() != eventNames().clickEvent && mouseEvent.type() != eventNames().auxclickEvent && pointerCaptureController.preventsCompatibilityMouseEventsForIdentifier(platformEvent.pointerId()))
```

Comment:
Should we make a helper function that checks for both `click` and `auxclick` so we don’t have to make the expression longer in each place? I see 15 places we could use it.

Reply:
definitely; would a helper method called `isClickOrAuxClickEvent` in `Event.h` be good? Or is there a better place for it?

Reply:
(I ended up going with `static bool isClickOrAuxClickEvent(const Event&);` on MouseEvent.h